README.md.tt

```diff
@@ -97,13 +97,8 @@ docker-compose -f docker-compose.test.yml run test bin/bundle exec spec [rspec-p
 ```
 
 ### Automated Code Review Setup
+- Add a bot (i.e. `team-nimblehq`) to this repository or to the organization. The bot requires permission level “Write” to be able to set a PR’s status.
 
-* Make sure that the config file `pronto.yml` for pronto has been added as a configuration file.
-
-* In the CI build script, add the following command:
-
-```sh
-if ([ $BRANCH_NAME != 'master' ] && [ $BRANCH_NAME != 'development' ]); then \
-  (echo "Running pronto"; bundle exec pronto run -f bitbucket_pr -c origin/development);  else \ (echo "Escaping pronto"); \
-fi
-```
+- Create a [Personal Access Token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
+from bot account with `public_repo` scope, and use it to as `DANGER_GITHUB_API_TOKEN
+` secret on Semaphore Environment Settings.
```

Comment:
it seems the ` secret on Semaphore Environment Settings.` is on a different line, let move it to the same line as the `and use it to as `DANGER_GITHUB_API_TOKEN` @rafayetn 

Another thing, not sure about the `use it to as` grammar, could we rephase it to just `and set it as DANGER_GITHUB_API_TOKEN on Semaphore`

Reply:
Changed!